vendor/assets/stylesheets/sn-icon-font.css

```diff
@@ -245,15 +245,6 @@
 .sn-icon-files:before {
   content: "\e948";
 }
-.sn-icon-file-word:before {
-  content: "\e949";
-}
-.sn-icon-file-powerpoint:before {
-  content: "\e94a";
-}
-.sn-icon-file-excel:before {
-  content: "\e94b";
-}
```

Comment:
Better ask Bor to prepare new font build without them, otherwise on next font update they will comeback.

Reply:
we can keep them, it will be anyway overwritten by office.scss file